clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -498,6 +499,20 @@ def _is_unschedulable(self, pod: KubernetesPod) -> bool:
                 return True
         return False
 
+    def get_node_priority(self, node_id: str) -> float:
+        pods = self._pods_by_ip[node_id]
+        if not pods or len(pods) == 0:
+            return 0.0
+        max_requested_resource = max([self.get_pod_resources_score(pod) for pod in pods])
+        # nodes have bigger pods = higher priority = lower possibility for choosing termination
+        return max_requested_resource
+
+    def get_pod_resources_score(self, pod: KubernetesPod) -> float:
+        resource = total_pod_resources(pod)
+        cpus = getattr(resource, "cpus") * 2
+        mem = getattr(resource, "mem") / 1000
```

Comment:
is there any possibility of unit confusion here with human readable unit input like 1 MiB etc?

Reply:
We are keeping mem with MB only.